docs/src/main/paradox/orientdb.md

```diff
@@ -0,0 +1,120 @@
+# OrientDB Connector
+
+The OrientDB connector provides Akka Stream sources and sinks for OrientDB.
+
+For more information about OrientDB please visit the [official documentation](http://orientdb.com/orientdb/).
+
+## Artifacts
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-stream-alpakka-orientdb_$scalaBinaryVersion$
+  version=$version$
+}
+
+## Usage
+
+Sources, Flows and Sinks provided by this connector need dbUrl & credentials to access to OrientDB.
+
+Scala
+: @@snip ($alpakka$/orientdb/src/test/scala/akka/stream/alpakka/orientdb/OrientDBSpec.scala) { #init-settings }
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #init-settings }
+
+We will also need an @scaladoc[ActorSystem](akka.actor.ActorSystem) and an @scaladoc[ActorMaterializer](akka.stream.ActorMaterializer).
+
+Scala
+: @@snip ($alpakka$/orientdb/src/test/scala/akka/stream/alpakka/orientdb/OrientDBSpec.scala) { #init-mat }
+
+Java
+: @@snip ($alpakka$/orientdb/src/test/java/akka/stream/alpakka/orientdb/OrientDBTest.java) { #init-mat }
+
+This is all preparation that we are going to need.
+
+### ODocument message

```

Comment:
Put the sections on level '##' and they'll show in the right-hand side overview.